doc/nrf/gs_installing.rst

```diff
@@ -279,11 +286,7 @@ To clone the repositories, complete the following steps:
 
       west init -m https\://github.com/nrfconnect/sdk-nrf --mr |release|
 
-   To check out the latest state of development, enter the following command::
-
-     west init -m https://github.com/nrfconnect/sdk-nrf --mr main
-
-   .. west-error-start
+      .. west-error-start
```

Comment:
Not sure if there is any edit here

Reply:
Just moved this part a bit earlier, that was the only change